scripts/dev_tools/pr_review.py

```
@@ -0,0 +1,198 @@
+    get_ollama_endpoint,
+    get_ollama_model,
+    validate_ollama_connection,
+)
+from review_common import build_prompt, emit_empty_diff_notice, finalize_review, truncate_diff
+
+
+def get_repo_info() -> tuple[str, str]:
+    """Extract owner and repo from git remote origin."""
+    try:
+        result = subprocess.run(
+            ["git", "config", "--get", "remote.origin.url"],
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)